src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -831,7 +831,7 @@ subroutine ocn_vmix_init(domain, err)!{{{
       logical, pointer :: config_use_implicit_bottom_drag
       real (kind=RKIND), pointer :: config_implicit_bottom_drag_coeff
       logical, pointer :: config_Rayleigh_friction, config_Rayleigh_bottom_friction, &
-                          config_Rayleigh_damping_coeff_depth_variable
```

Comment:
@pwolfram All changes in mpas_ocn_vmix.F look like a mistake. Could you revert this file so no changes are made?

Reply:
@mark-petersen, this was a typo from before.  I'll fix this in a separate PR for now because it is really a separate change.  Thanks for catching this issue.

Reply:
Looks good, thanks.

Reply:
Fixed via #354